.circleci/run-build-locally.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+curl --user ${CIRCLE_CI_TOKEN}: \
+    --request POST \
+    --form revision=403b42af280ec3023a06826f4edaceda8a1f8a2d\
```

Comment:
should this be a token?

Reply:
This is only for local debugging.  Each time someone wanted to debug locally they would update the revision value with the git log id.  I don't think it should be a token as that would be more difficult to manage.